linera-storage/src/db_storage.rs

```diff
@@ -225,55 +225,58 @@ pub mod metrics {
     });
 }
 
-/// The default key used when the root_key contains the information.
-const DEFAULT_KEY: &[u8] = &[0];
+/// The key used for blobs. The Blob ID itself is contained in the root key.
+const BLOB_KEY: &[u8] = &[42];
 
-/// The second key used when the root_key contains the information.
-const ONE_KEY: &[u8] = &[1];
+/// The key used for blob states. The Blob ID itself is contained in the root key.
+const BLOB_STATE_KEY: &[u8] = &[49];
 
-fn get_01_keys() -> Vec<Vec<u8>> {
-    vec![vec![0], vec![1]]
+/// The key used for lite certificates. The cryptohash itself is contained in the root key.
+const LITE_CERTIFICATE_KEY: &[u8] = &[91];
+
+/// The key used for confirmed blocks. The cryptohash itself is contained in the root key.
+const BLOCK_KEY: &[u8] = &[221];
+
+/// The key used for the network description.
+const NETWORK_DESCRIPTION_KEY: &[u8] = &[119];
```

Comment:
Where do all those numbers come from?

Reply:
They have been chosen somewhat at random.

Reply:
Is there a reason to not just use 0, 1, 2, …?